lib/models/CartoonCat.js

```diff
@@ -1,4 +1,29 @@
 const pool = require('../utils/pool');
 module.exports = class CartoonCat {
   // TODO -- fill in this class!
+  constructor(row) {
+    this.id = row.id;
+    this.name = row.name;
+    this.type = row.type;
+    this.lives = row.lives;
+    this.url = row.url;
+    this.year = row.year;
+    this.isSidekick = row.is_sidekick;
+  }
+
+  static async getAll() {
+    const { rows } = await pool.query('SELECT * from cartoon_cats');
+    return rows.map((catRow) => new CartoonCat(catRow));
+  }
+
+  static async getById(id) {
+    // rows
+    //select all cats
+    // select
+    const { rows } = await pool.query(
+      'SELECT * from cartoon_cats WHERE id = $1',
```

Comment:
generally it's all caps except for table / column names (to help those stand out from the SQL language or no caps so like:
```SELECT * FROM cartoon_cats WHERE id = $1```
or
```select * from cartoon_cats where id = $1```